skbio/io/_exception.py

```diff
@@ -84,6 +84,11 @@ class QSeqFormatError(FileFormatError):
     pass
 
 
+class StockholmFormatError(FileFormatError):
+    """Raised when a ``stockholm`` formatted file cannot be parsed."""
+    pass
```

Comment:
`pass` shouldn't be necessary as long as there's a docstring. If it's not affecting coverage though I'd leave it in since it's more explicit.